.github/workflows/release.yml

```
@@ -60,8 +60,75 @@ jobs:
     runs-on: ubuntu-latest
     needs: publish_binary
     steps:
-      - name: Publish SDKs
-        uses: pulumi/pulumi-package-publisher@v0.0.14
+      - name: Checkout Repo
+        uses: actions/checkout@v2
+
+      - name: Unshallow clone for tags
+        run: git fetch --prune --unshallow --tags
+
+      - name: Install Go
+        uses: actions/setup-go@v3
+        with:
+          go-version: ${{ matrix.goversion }}
+
+      - name: Install pulumictl
+        uses: jaxxstorm/action-install-gh-release@v1.10.0
+        with:
+          repo: pulumi/pulumictl
+
+      - name: Install pulumi
+        uses: pulumi/actions@v4
+
+      - name: Setup Node
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{matrix.nodeversion}}
+          registry-url: ${{env.NPM_REGISTRY_URL}}
+
+      - name: Setup DotNet
+        uses: actions/setup-dotnet@v1
+        with:
+          dotnet-version: ${{matrix.dotnetverson}}
```

```suggestion
          dotnet-version: ${{ matrix.dotnetverson }}
```
To follow the code style :)